ref/appliance-command-reference/ams.md

```diff
@@ -0,0 +1,21 @@
+The `ams` command provides access to the Anbox Management Service (AMS) through subcommands. It exposes the AMS HTTPS service on the public endpoint of the machine on which the appliance is running. Using this command, you can [control AMS remotely](https://discourse.ubuntu.com/t/managing-ams-access/17774).
```

Comment:
, you can [control AMS remotely](https://discourse.ubuntu.com/t/managing-ams-access/17774).
->
, you can expose or unexpose an AMS service through the load balancer. 

The above [link](remote control to AMS) has nothing to do with service exposure